Libraries/Components/Button.windows.js

```diff
@@ -124,12 +177,63 @@ class Button extends React.Component {
         testID={testID}
         disabled={disabled}
         onPress={onPress}>
-        <View style={buttonStyles}>
-          <Text style={textStyles} disabled={disabled}>{formattedTitle}</Text>
-        </View>
+          {content}
       </Touchable>
     );
   }
+
+  _onKeyDown = (e): void => {
+    if (!this.props.disabled) {
+      if (this.props.onPress) {
+        var key = e.nativeEvent.key;
+        // ENTER triggers press on key down
+        if (key === KEY_CODE_ENTER) {
+          this.props.onPress(e);
+          return;
```

Comment:
>return; [](start = 10, length = 7)

Why do we have a `return` here for `_onKeyDown` and `_onKeyUp`?

Reply:
Those were beefier before. I'll remove the useless.

Reply:
Those were beefier before. I'll remove the useless.

---
In reply to: [156557386](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r156557386) [](ancestors = 156557386)